ChaosMod/Effects/db/Misc/MiscUTurn.cpp

```diff
@@ -36,10 +38,31 @@ static void OnStart()
 	SET_GAMEPLAY_CAM_RELATIVE_HEADING(camHeading);
 }
 
+// effect by Reguas
+
+static void OnStartFake()
+{
+	OnStart();
+
+	GetComponent<EffectDispatcher>()->OverrideEffectNameId("misc_fakeuturn", "misc_uturn");
+
+	WAIT(g_Random.GetRandomInt(6000, 9000));
+
+	OnStart();
+}
+
 // clang-format off
 REGISTER_EFFECT(OnStart, nullptr, nullptr, EffectInfo
 	{
 		.Name = "U-Turn",
 		.Id = "misc_uturn"
 	}
+);
```

Comment:
```suggestion
);
// clang-format on
```
Doesn't matter in this case but for uniformity